src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -146,3 +149,6 @@ def save(self):
 
     def run(self):
         pass
+
+    def on_leave(self, *args):
```

Comment:
Is a matching on_enter needed to load the temp config?

Reply:
No, on init the dwt config module will load it if it exists.

Reply:
ah but what if the user goes in and out of the app? surely init will only run at start up?

Reply:
If the screen is disposed, when the screen is created __init__ will be run again. If the screen isn't disposed, the object will still be there.